pkgs/by-name/do/dovecot_exporter/2_3.nix

```
@@ -0,0 +1,7 @@
+{
+  callPackage,
+  dovecot_2_4,
+}@args:
+callPackage ./generic.nix args {
+  dovecot = dovecot_2_4;
```

Should this be 2_3? 🤔 
```suggestion
  dovecot_2_3,
}@args:
callPackage ./generic.nix args {
  dovecot = dovecot_2_3;
```

---

yup, thanks!